strapi/strapi
# [DEITS] - Add remote transfer capabilities (push)

### What does it do?

- Add a websocket server on an /admin/transfer route to handle remote transfers
- Add a strapi remote destination provider that sends data through the /admin/transfer route

### Why is it needed?

It is need to enable data transfers on remote (and running) strapi instances

### How to test it?

- Run a first strapi instance
- Go to another strapi instance and run `yarn strapi transfer --to "ws://localhost:1337/admin/transfer"`
- It should transfer data from your current local instance to the one running on localhost:1337
